src/GafferScene/EditScopeAlgo.cpp

```diff
@@ -165,7 +166,10 @@ SceneProcessorPtr transformProcessor()
 		plug->setInput( spreadsheet->outPlug()->getChild<Plug>( name ) );
 	}
 
-	PlugAlgo::promoteWithName( spreadsheet->rowsPlug(), "edits" );
+	auto rowsPlug = static_cast<Spreadsheet::RowsPlug *>( PlugAlgo::promoteWithName( spreadsheet->rowsPlug(), "edits" ) );
+	Metadata::registerValue( rowsPlug, "spreadsheet:defaultRowVisible", new BoolData( false ) );
+	Metadata::registerValue( rowsPlug->defaultRow(), "spreadsheet:rowNameWidth", new IntData( 300 ) );
```

Comment:
This is a great improvement, but we def need to fix the default width of NodeEditors now, as it looks like this when you use the navigation menu:

![image](https://user-images.githubusercontent.com/896779/77410780-5a503180-6db3-11ea-8332-ccd47581476b.png)

Reply:
Totally - I'll follow up with a PR to take care of this shortly.